js_modules/dagster-ui/packages/ui-core/src/partitions/OpJobPartitionsView.tsx

```diff
@@ -413,11 +415,10 @@ export const OpJobPartitionsViewContent = React.memo(
           <Subheading>Backfill history</Subheading>
         </Box>
         <Box margin={{bottom: 20}}>
-          <JobBackfillsTable
-            partitionSetName={partitionSet.name}
-            repositorySelector={repositorySelector}
-            partitionNames={partitionNames}
-            refetchCounter={backfillRefetchCounter}
+          <RunsFeedTableWithFilters
+            hideTags={BACKFILL_TAGS}
+            filter={{pipelineName: partitionSet.pipelineName}}
+            view={RunsFeedView.BACKFILLS}
```

Comment:
Should we just automatically hide backfill tags if the view is RunsFeedView.BACKFILLS?